diff --git a/src/lib.rs b/src/lib.rs
index 7f09187..48df67e 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -570,7 +570,7 @@ mod internal {
                     str_to_nsstr("AVCaptureDeviceTypeBuiltInTrueDepthCamera")
                 }
                 AVCaptureDeviceType::ExternalUnknown => {
-                    str_to_nsstr("AVCaptureDeviceTypeBuiltInExternalUnknown")
+                    str_to_nsstr("AVCaptureDeviceTypeExternalUnknown")
                 }
             }
         }
